site/docs/guide/api.md

```diff
@@ -132,3 +183,99 @@ async function sendHelloTo12345() {
 ```
 
 Basically, all parameters of the function signature are merged with the options object when you use the raw API.
+
+## Choosing a Data Center Location
+
+> [Skip](./filter-queries) the rest of the page if you are just getting started.
+
+If you want to reduce the networking latency of your bot, it matters where you host it.
+
+The Bot API server behind `api.telegram.org` is hosted in Amsterdam in the Netherlands.
+Therefore, the best location to run your bot is Amsterdam.
+
+::: tip Hosting Comparison
+You may be interested in our [comparison on hosting providers](../hosting/comparison#comparison-tables).
+:::
+
+However, there might be an even better place to run your bot, although this takes significantly more effort.
+
+[Remember](#general-information) that the Bot API server does not actually contain your bot.
+It only relays requests, translates between HTTP and MTProto, and so on.
+The Bot API server may be in Amsterdam, but the Telegram servers are distributed across three locations: Amsterdam, Miami, and Singapore.
+
+Thus, when the Bot API server sends a request to the Telegram servers, it may have to send data halfway around the globe.
+Whether or not this happens depends on the data center of the bot itself.
+The data center of the bot is the same data center as for the user who created the bot.
+The data center of a user depends on many factors, including the location of the user.
+
+Hence, this is what you can do if you want to reduce the latency even further.
+
+1. Contact [@where_is_my_dc_bot](https://t.me/where_is_my_dc_bot) and send a file that was uploaded with your own account.
+   It will tell you the location of your user account.
+   This is also the location of your bot.
```

Comment:
We said otherwise

Reply:
What do you mean?

Reply:
![image](https://github.com/grammyjs/website/assets/49933115/2106f253-b94a-4028-a10b-30dd97007241)

Reply:
![image](https://github.com/grammyjs/website/assets/49933115/c679bd63-2081-4961-888e-9df3b71a1ef0)

Reply:
I don't understand what you're trying to say. It's still the case that if you create a bot, it will default to residing in the same DC as your user account, so that's the reasonable thing to document. What are you suggesting?

Reply:
@KnorpelSenf, @roj1512, is this question considered closed or what?

Reply:
I don't know what the question is so I'll leave this to @roj1512 :D

Reply:
Resolve.